src/webui/www/private/scripts/misc.js

```diff
@@ -43,10 +43,13 @@ window.qBittorrent.Misc = (function() {
             escapeHtml: escapeHtml,
             safeTrim: safeTrim,
             toFixedPointString: toFixedPointString,
-            containsAllTerms: containsAllTerms
+            containsAllTerms: containsAllTerms,
+            MAX_ETA: 8640000
         };
     };
 
+Object.freeze(window.qBittorrent.Misc);
```

Comment:
I see why it isn't working for you. This should be placed at the end of this file.

Reply:
Thanks fixed